HUnit.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version: 1.12
 
--- This file has been generated from package.yaml by hpack version 0.34.3.
+-- This file has been generated from package.yaml by hpack version 0.36.0.
```

Comment:
Apparently the repository is missing a `.gitattributes` file, something like
https://github.com/sol/markdown-unlit/blob/main/.gitattributes.

If you wanna add that, then that would be more than welcome.